### PR TITLE
Add Language: Sourcepawn

### PR DESCRIPTION
**Pull request by SammCheese**
Adds Support for the Language [Sourcepawn](https://wiki.alliedmods.net/Introduction_to_SourcePawn_1.7)

### CODE REVIEW

File: languages/index.js [hunk 1]
```diff
@@ -1,730 +1,736 @@
...
-  },
-  {
-    name: 'Wenyan',
-    id: '文言',
-    scopeName: 'source.wenyan',
-    grammar: loadJSON('文言.tmLanguage.json'),
-    aliases: ['wenyan']
-  }
-]
+const fs = require('fs')
```
Comment (Vap0r1ze) [inline languages/index.js]: Your IDE made the whole file CRLF line endings, can you revert this back to LF before I merge

**Review by Vap0r1ze — CHANGES_REQUESTED** [review]

File: languages/index.js (hunk 1 above)
Comment (SammCheese) [reply languages/index.js]: Changed to LF

**Review by Vap0r1ze — APPROVED** [review]
Yea looks perfect now

**Comment by Vap0r1ze** [comment]
Thanks for contributing!

**Pull request merged**